src/networking.rs

```diff
@@ -14,14 +17,120 @@ pub const RESP_TIMEOUT: Duration = Duration::from_secs(10);
 pub const ATTESTATION_PROPAGATION_SLOT_RANGE: usize = 32;
 pub const MAXIMUM_GOSSIP_CLOCK_DISPARITY: Duration = Duration::from_millis(500);
 
-pub use multiaddr::Multiaddr;
+/// Public keys with byte-lengths smaller than `MAX_INLINE_KEY_LENGTH` will be
+/// automatically used as the peer id using an identity multihash.
+const MAX_INLINE_KEY_LENGTH: usize = 42;
 
-// TODO see how much this can be tightened
-const MAX_PEER_ID_BYTE_COUNT: usize = 512;
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
```

Comment:
let's move these so they are grouped w/ the rest of the imports

Reply:
done!